Inaccurate log messege when push accessory fail

When the push accessory failed, it got the following error message
```
2024-06-11T07:13:13Z [INFO] [/pkg/scan/job.go:232]: Get report for mime type: application/vnd.security.sbom.report+json; version=1.0
2024-06-11T07:13:15Z [ERROR] [/pkg/scan/sbom/sbom.go:108]: error when create accessory from image could not parse reference: http://core:8080/sample/alpine@sha256:f71371ca77c9bf486c31376fc07b79c7c6e33f60806b7c4d9cb9acac3b552189
2024-06-11T07:13:15Z [ERROR] [/pkg/scan/job.go:310]: Failed to convert vulnerability data to new schema for report dcad060f-4ec9-40b4-a2b0-3da2e2fcb993, error could not parse reference: http://core:8080/sample/alpine@sha256:f71371ca77c9bf486c31376fc07b79c7c6e33f60806b7c4d9cb9acac3b552189
```

--- a/src/controller/artifact/model.go
+++ b/src/controller/artifact/model.go
@@ -80,6 +80,7 @@ func (artifact *Artifact) SetAdditionLink(addition, version string) {
 	artifact.AdditionLinks[addition] = &AdditionLink{HREF: href, Absolute: false}
 }
 
+// SetSBOMAdditionLink set the link of SBOM addition
 func (artifact *Artifact) SetSBOMAdditionLink(sbomDgst string, version string) {
 	if artifact.AdditionLinks == nil {
 		artifact.AdditionLinks = make(map[string]*AdditionLink)
@@ -88,7 +89,7 @@ func (artifact *Artifact) SetSBOMAdditionLink(sbomDgst string, version string) {
 	projectName, repo := utils.ParseRepository(artifact.RepositoryName)
 	// encode slash as %252F
 	repo = repository.Encode(repo)
-	href := fmt.Sprintf("/api/%s/projects/%s/repositories/%s/artifacts/%s/additions/%s", version, projectName, repo, sbomDgst, addition)
+	href := fmt.Sprintf("/api/%s/projects/%s/repositories/%s/artifacts/%s/additions/sbom", version, projectName, repo, sbomDgst)
 
 	artifact.AdditionLinks[addition] = &AdditionLink{HREF: href, Absolute: false}
 }

--- a/src/pkg/scan/job.go
+++ b/src/pkg/scan/job.go
@@ -304,7 +304,7 @@ func (j *Job) Run(ctx job.Context, params job.Parameters) error {
 
 		reportData, err := handler.PostScan(ctx, req, rp, rawReports[i], startTime, robotAccount)
 		if err != nil {
-			myLogger.Errorf("Failed to convert vulnerability data to new schema for report %s, error %v", rp.UUID, err)
+			myLogger.Errorf("handler failed at PostScan, report %s, error %v", rp.UUID, err)
 			return err
 		}
 

